packages/flutter_tools/lib/src/web/chrome.dart

```diff
@@ -490,6 +490,10 @@ class Chromium {
         if (i == attempts) {
           rethrow;
         }
+      } on IOException catch (_) {
```

Comment:
```suggestion
      } on IOException {
```

I made a suggestion, so we can smoke test the CLABot thing

Reply:
Did you? I still see the CLA check as green (the bot shouldn't break until I accept the suggestion, if at all).

Reply:
Oh I misread this. Sure lets try it

Reply:
Looks like it still passed :)